docs/guides/network-profiler.md

```diff
@@ -0,0 +1,170 @@
+
+- Open Unity's Profiler window `Window > Analysis > Profiler`.
+- In the Profiler window, click on the `Profiler Modules` dropdown.
+- Select `Normcore` from the available profiler modules.
+
+<video width="100%" controls><source src={profilerSetUp} /></video> 
+
+### Profiler data
+
+With the Normcore profiler active, you'll see two key metrics in the Profiler window:
+- **Sent**: Data being sent from the client over time.
+- **Received**: Data being received by the client over time.
+
+## Profiling
+
+Pay attention to the graphs for sent and received data. Look for spikes and trends correlating with specific events or actions. For instance, if you see a jump in data when new rigidbody instances are created, those instances are likely contributing heavily to traffic.
+
+This video shows how sent data increases over time as new instances are created. The profiler reveals a pattern where each instance adds to the cumulative sent data, causing spikes.
+
+<video width="100%" controls><source src={profileUnlimitedInstances} /></video> 
+
+## Solution
+
+To optimize bandwidth, it's crucial to manage rigidbody instances effectively. Two common strategies are reusing instances and destroying those that are too far from the player or no longer contributing to the simulation. Implementing these strategies can significantly reduce transmitted data.
+
+In our sample, we have implemented a pooling mechanism that will create instances up to a certain limit and reuse the oldest instance when the instance limit is reached.
+
+```csharp
+using System.Collections.Generic;
+using Normal.Realtime;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class ShooterOptimized : MonoBehaviour {
+    // A queue to store basketballs
+    private Queue<GameObject> _basketballPool = new Queue<GameObject>();
+
+    // Called once per frame
+    private void Update() {
+        // Check if the left mouse button was pressed this frame
+        if (Mouse.current.leftButton.wasPressedThisFrame) ShootTenBasketballs();
+    }
+
+    // Shoot 10 basketballs
+    private void ShootTenBasketballs() {
+        for (var i = 0; i < 10; i++) Shoot();
+    }
+
+    // Shoot method to get an instance from the pool
+    private void Shoot() {
+        // Spawn a basketball in the same location as the shooter component
+        var basketball = InstantiateBasketballFromPool();
+        
+        // Set the velocity to a random forward/up velocity
+        basketball.GetComponent<Rigidbody>().velocity = basketball.transform.forward * 5f;
+        basketball.GetComponent<RealtimeTransform>().RequestOwnership();
+    }
+
+    // Instantiate a basketball from the pool
+    private GameObject InstantiateBasketballFromPool() {
+        GameObject basketball;
+
+        var position = transform.position + transform.right * Random.Range(-4f, 4f);
```

Comment:
I don't really understand why this moved from Shoot to here. I would expect Shoot to set the position/rotation/velocity/etc. Setting the spawn position in the pooling method doesn't make sense to me.

Reply:
I moved it here because Realtime.Instantiate needs the position/rotation for it to work well. I can pass the position as a parameter of InstantiateBasketballFromPool